src/System.Buffers.ReaderWriter/System/Buffers/Writer/OutputWriter.cs

```diff
@@ -6,31 +6,23 @@
 
 namespace System.Buffers.Writer
 {
-    public static class OutputWriter
-    {
-        public static OutputWriter<T> Create<T>(T output) where T : IBufferWriter<byte>
-        {
-            return new OutputWriter<T>(output);
-        }
-    }
-
-    public ref struct OutputWriter<T> where T : IBufferWriter<byte>
+    public ref partial struct BufferWriter<T> where T : IBufferWriter<byte>
     {
         private T _output;
         private Span<byte> _span;
         private int _buffered;
 
-        public OutputWriter(T output)
+        public BufferWriter(T output)
         {
             _buffered = 0;
             _output = output;
             _span = output.GetSpan();
         }
 
-        public Span<byte> Span => _span;
+        public Span<byte> Buffer => _span;
```

Comment:
Why `Buffer`? It would be better to name the property `Span` like it done in [`Memory`](https://github.com/dotnet/coreclr/blob/8bad66fca4442839adea749e7a35b0dda8023eda/src/mscorlib/shared/System/Memory.cs#L186).

Reply:
I thought buffer communicates better that the write needs to be Flushed (and one of the fields is called _buffered).